go.mod

```diff
@@ -7,33 +7,33 @@ require (
 	github.com/Microsoft/go-winio v0.4.15
 	github.com/aws/aws-sdk-go v1.35.24
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/ttrpc v1.0.2
+	github.com/containerd/ttrpc v1.1.0
 	github.com/containerd/typeurl v1.0.2
 	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v20.10.17+incompatible
+	github.com/docker/docker v20.10.18+incompatible
```

Comment:
k8s is on `v20.10.17`, do we plan to upgrade k8s to  `v20.10.18` as well?

https://github.com/kubernetes/kubernetes/blob/005a5ca08173b080193ecb01421e2513f32d045b/go.mod#L328

Reply:
@bobbypage yep!